packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2513,6 +2529,8 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   /// determine if a node is previous to this one.
   bool get isBlockingSemanticsOfPreviouslyPaintedNodes => false;
 
+  bool get isMergingSemanticsOfDescendants => false;
```

Comment:
you're trying to sneak an undocumented member into the codebase without me noticing! :-P

Reply:
It was an honest mistake! I swear! 😆 

On a more serious note: It looks like we've documented all but one public member in the Flutter code base. Can we document the last remaining one and make the analyzer fail if new undocumented members are added? Or is something blocking that step?